shell/lisa_shell

```diff
@@ -396,7 +396,7 @@ EOF
 
```

Comment:
While we are at it, I'm not sure calling `lisa-venv-activate` is really needed anymore in `lisa-wltest-init`, since that is already done when sourcing lisa_shell itself (bottom of the file). That probably dates back to before we revamped the install process.